src/parser/wsdl.js

```diff
@@ -418,6 +430,7 @@ class WSDL {
           wsdl = new WSDL(definition, uri, options);
           WSDL_CACHE[uri] = wsdl;
           wsdl.WSDL_CACHE = WSDL_CACHE;
+          wsdl.syncLoad = syncLoad
```

Comment:
Nitpick: Please use semicolons for your sentences in this file as well.

Reply:
done :-)